gr-fec/lib/reed-solomon/rs.3

```diff
@@ -62,12 +51,12 @@ and \fBdecode_rs_8\fR.
 
 The _8 and _ccsds functions do not require initialization.
 To use the general purpose RS encoder or decoder (i.e.,
-the _char or _int versions), the user must first
-call \fBinit_rs_int\fR or \fBinit_rs_char\fR as appropriate. The
+the _char ), the user must first
+call \fBinit_rs_char\fR as appropriate. The
 arguments are as follows:
 
 \fBsymsize\fR gives the symbol size in bits, up to 8 for \fBinit_rs_char\fR
-or 32 for \fBinit_rs_int\fR on a machine with 32-bit ints (though such a
+or 32 for on a machine with 32-bit ints (though such a
 huge code would exhaust memory limits on a 32-bit machine). The resulting
```

Comment:
This sentence is not correct now. Once we have removed the `_int` functions, the symbol size can only be 8 bits.